bindings/node.js/test/kzg.test.ts

```diff
@@ -53,32 +67,101 @@ const generateRandomBlob = (): Uint8Array => {
       return x;
     })
   );
-};
-
-const blobValidLength = randomBytes(BYTES_PER_BLOB);
-const blobBadLength = randomBytes(BYTES_PER_BLOB - 1);
-const commitmentValidLength = randomBytes(BYTES_PER_COMMITMENT);
-const commitmentBadLength = randomBytes(BYTES_PER_COMMITMENT - 1);
-const proofValidLength = randomBytes(BYTES_PER_PROOF);
-const proofBadLength = randomBytes(BYTES_PER_PROOF - 1);
-const fieldElementValidLength = randomBytes(BYTES_PER_FIELD_ELEMENT);
-const fieldElementBadLength = randomBytes(BYTES_PER_FIELD_ELEMENT - 1);
+}
 
+/**
+ * Converts hex string to binary Uint8Array
+ *
+ * @param {string} hexString Hex string to convert
+ *
+ * @return {Uint8Array}
+ */
 function bytesFromHex(hexString: string): Uint8Array {
   if (hexString.startsWith("0x")) {
     hexString = hexString.slice(2);
   }
   return Uint8Array.from(Buffer.from(hexString, "hex"));
 }
 
-function bytesEqual(a: Uint8Array | Buffer, b: Uint8Array | Buffer): boolean {
+/**
+ * Verifies that two Uint8Arrays are bitwise equivalent
+ *
+ * @param {Uint8Array} a
+ * @param {Uint8Array} b
+ *
+ * @return {void}
+ *
+ * @throws {Error} If arrays are not equal length or byte values are unequal
+ */
+function bytesEqual(a: Uint8Array | Buffer, b: Uint8Array | Buffer): void {
```

Comment:
```suggestion
function assertEqual(a: Uint8Array | Buffer, b: Uint8Array | Buffer): void {
```
if we throw then its better to rename it as `assert` as its semantics are more clear that way

Reply:
Updated.

Reply:
Updated.